bus-mapping/src/evm/opcodes/callop.rs

```diff
@@ -25,6 +25,12 @@ impl<const N_ARGS: usize> Opcode for CallOpcode<N_ARGS> {
         let geth_step = &geth_steps[0];
         let mut exec_step = state.new_step(geth_step)?;
 
+        // Callee address of stack is used to check if callee account exists in access
+        // list and get code hash. Since the below `call.address` is set to
+        // caller address in `state.parse_call` function for opcode
+        // `DELEGATECALL`. See `CircuitInputStateRef::parse_call` for details.
+        let callee_address = geth_step.stack.nth_last(1)?.to_address();
```

Comment:
I remember `current_call.address` stands for callee address as in line L58

Reply:
Fixed to get callee address (code address) from `call.code_source` as:
```
let callee_address = match call.code_source {
    CodeSource::Address(address) => address,
    _ => call.address,
};
```